docs/output-files/txt-histogram-state-populations.md

```diff
@@ -0,0 +1,157 @@
+---
+layout: default
+title: (*.txt) State populations from Histogram analysis
```

Comment:
To reduce the number of output files with the same file extension .txt, we could use a combination of previous file extensions, such as hist stats for state populations from histogram analysis. 

But this might be too much ... leave as is ... I am only thinking loudly

Reply:
If you want my opinion, many file extensions is not good. It doesn't matter if several files have the same extension, especially text files. Also, file name are always appended with an appropriate extension not to confuse the user.